libs/sdk-core/src/models.rs

```diff
@@ -1238,8 +1238,8 @@ pub struct SwapInfo {
 
     /// bolt11 invoice to claim the sent funds.
     pub bolt11: Option<String>,
-    /// Amount of sats payed to the claim address.
-    pub paid_sats: u64,
+    /// Amount of millisatoshis paid to the claim address.
+    pub paid_msat: u64,
```

Comment:
SDK requests / responses refering to LN amounts are using `msat`.

Those refering to onchain amounts are now using `sat`, for example the other `SwapInfo` fields

https://github.com/breez/breez-sdk/blob/1cffacb3e855360ad0e87fc93397c9f6c52f0585/libs/sdk-core/src/models.rs#L1243-L1246

or `ReverseSwapInfo`

https://github.com/breez/breez-sdk/blob/1cffacb3e855360ad0e87fc93397c9f6c52f0585/libs/sdk-core/src/models.rs#L160-L163

Wouldn't it then make more sense to keep using `sat` here instead of `msat`?

Reply:
In the issue it was suggested that the paid amount (over lightning) should be stored as millisatoshis

Reply:
You're right.

> Amount of millisatoshis paid to the claim address
> pub paid_msat: u64

Its not clear if this means
- "amount which left the LN wallet" (LN, should be msat) or
- "amount which arrived in the claim address" (onchain, should be sat)

Turns out its the former. The swap field is written in `update_swap_paid_amount` as the LN invoice msat amount:

https://github.com/breez/breez-sdk/blob/be1915da8bf066042f5adc686112cf229f55f0fe/libs/sdk-core/src/swap_in/swap.rs#L118-L126

Reply:
>    Amount of millisatoshis claimed from sent funds and paid for via bolt11 invoice.
>    pub paid_msat: u64,

Improved the field description [9ba9dcf](https://github.com/breez/breez-sdk/pull/766/commits/9ba9dcf5eb4a1e8f2ce580afc74ea070f01fb2fc)

Reply:
Thanks, that's indeed clearer.